packages/examples/cvat/exchange-oracle/src/crons/_cron_job.py

```diff
@@ -0,0 +1,71 @@
+import inspect
+import logging
+from collections.abc import Callable
+from functools import wraps
+from typing import NamedTuple
+
+from sqlalchemy.orm import Session
+
+from src.db import SessionLocal
+from src.log import get_logger_name
+
+
+class CronSpec(NamedTuple):
+    manage_session: bool
+    repr: str
+
+
+def _validate_cron_function_signature(fn: Callable[..., None]) -> CronSpec:
+    cron_repr = repr(fn.__name__)
+    parameters = dict(inspect.signature(fn).parameters)
+
+    session_param = parameters.pop("session", None)
+    if session_param is not None and session_param.annotation is not Session:
```

Comment:
Can it be a session subclass?

Reply:
No, see https://github.com/humanprotocol/human-protocol/pull/2428#discussion_r1740695273